src/main/perf/LineFileDocs.java

```diff
@@ -584,11 +589,11 @@ public Document nextDoc(DocState doc) throws IOException {
     if (addDVFields) {
       doc.titleBDV.setBytesValue(new BytesRef(title));
       doc.titleDV.setBytesValue(new BytesRef(title));
-      doc.titleTokenized.setStringValue(title);
       doc.monthDV.setBytesValue(new BytesRef(months[doc.dateCal.get(Calendar.MONTH)]));
       doc.dayOfYearDV.setLongValue(doc.dateCal.get(Calendar.DAY_OF_YEAR));
       doc.dayOfYearIP.setIntValue(doc.dateCal.get(Calendar.DAY_OF_YEAR));
     }
+    doc.titleTokenized.setStringValue(title);
```

Comment:
Whoa, a pre-existing bug where we were failing to set the title if DV were disabled!?